x-pack/legacy/plugins/siem/public/components/timeline/body/helpers.ts

```diff
@@ -93,5 +95,6 @@ export const getColumnHeaders = (
 };
 
 /** Returns the (fixed) width of the Actions column */
-export const getActionsColumnWidth = (isEventViewer: boolean): number =>
-  isEventViewer ? EVENTS_VIEWER_ACTIONS_COLUMN_WIDTH : DEFAULT_ACTIONS_COLUMN_WIDTH;
+export const getActionsColumnWidth = (isEventViewer: boolean, showCheckboxes = false): number =>
```

Comment:
consider taking named parameters here, such that applications of this function, which currently look like `getActionsColumnWidth(true, true)` become `getActionsColumnWidth({ isEventViewer: true, showCheckboxes: true})`

Reply:
Good call 👍I've made a note to rework that in `Part II` as I'll need to update this logic to support any number of arbitrary actions that can be injected by the parent component.